node/rpc/src/common_api.rs

```diff
@@ -2,17 +2,17 @@
 // SPDX-License-Identifier: Apache-2.0, MIT
 
 use jsonrpc_v2::Error as JsonRpcError;
-use std::convert::TryInto;
 
-use fil_types::build_version::{user_version, APIVersion, Version, RUNNING_NODE_TYPE};
+use fil_types::build_version::{APIVersion, Version};
 use rpc_api::common_api::*;
 
-pub(crate) async fn version(block_delay: u64) -> Result<VersionResult, JsonRpcError> {
-    #[allow(clippy::needless_borrow)]
-    let v: Version = (&*RUNNING_NODE_TYPE.read().await).try_into()?;
+pub(crate) async fn version(
+    block_delay: u64,
+    forest_version: &'static str,
+) -> Result<VersionResult, JsonRpcError> {
     Ok(APIVersion {
-        version: user_version().await,
-        api_version: v,
+        version: forest_version.to_string(),
+        api_version: Version::new(0, 0, 0),
```

Comment:
We can use the version of the `rpc` library here?

Reply:
Sure! Let's do this.

Reply:
Or maybe pass in the forest version like you do with the static string.

Reply:
@david Are you suggesting that we use the `semver` crate?

Reply:
Yes.